Image-Processing/BFR_Sample_Rest.ipynb

```diff
@@ -0,0 +1,576 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Azure Cognitive Search sample \n",
+    "## Passing Images as Binary File References\n",
+    "\n",
+    "Cognitive Search skillsets that need to pass images to custom skills use a binary file reference to serialize the images to pass them to and from skills. This sample demosntrates an example of how skills can be configured to accept an image as an input from the skillset and return images as outputs to the skillset. This example does nothing more than segment an image based on the layout from OCR. The sole purpose of this sample is to demosntrate how you pass images to skills and how skills can return images.\n",
```

Comment:
Line 10 -- two typos on "demonstrate" in this paragraph

Can explain why the REST API used here?  You could put this in the intro or in the prerequisites:  "This sample uses Python and the Search REST API (the Azure SDK for Python does not support knowledge store definitions at this time)."

Reply:
I'd rather not call that out as we would need to go back and change that when the knowledge store is supported.